benefit-finder/cypress/e2e/death-of-a-loved-one/selected-criteria-eligibility-benefits.cy.js

```diff
@@ -98,4 +114,74 @@ describe('Validate correct eligibility benefits display based on selected criter
       )
   })
+
+  it('qa scenario 1 Covid EN - Verify correct benefit results for query values that includes covid in search parameter of URL', () => {
+    const benefitsCriteria = BENEFITS_ELIBILITY_DATA.scenario_1_covid.en.param
+    const applicant_date_of_birth = encodeURI(
+      `{"month":"${
+        benefitsCriteria.applicant_date_of_birth_month - 1
+      }","day":"${benefitsCriteria.applicant_date_of_birth_day}","year":"${
+        benefitsCriteria.applicant_date_of_birth_year
+      }"}`
+    )
+    const deceased_date_of_death = encodeURI(
+      `{"month":"${benefitsCriteria.deceased_date_of_death_month - 1}","day":"${
+        benefitsCriteria.deceased_date_of_death_day
+      }","year":"${benefitsCriteria.deceased_date_of_death_year}"}`
+    )
+    cy.visit(
+      `/iframe.html?args=&id=app--primary&viewMode=story&applicant_date_of_birth=${applicant_date_of_birth}&applicant_relationship_to_the_deceased=${benefitsCriteria.applicant_relationship_to_the_deceased}&applicant_marital_status=${benefitsCriteria.applicant_marital_status}&applicant_citizen_status=${benefitsCriteria.applicant_citizen_status}&applicant_care_for_child=${benefitsCriteria.applicant_care_for_child}&applicant_paid_funeral_expenses=${benefitsCriteria.applicant_paid_funeral_expenses}&deceased_date_of_death=${deceased_date_of_death}&deceased_death_location_is_US=${benefitsCriteria.deceased_death_location_is_US}&deceased_paid_into_SS=${benefitsCriteria.deceased_paid_into_SS}&deceased_public_safety_officer=${benefitsCriteria.deceased_public_safety_officer}&deceased_miner=${benefitsCriteria.deceased_miner}&deceased_american_indian=${benefitsCriteria.deceased_american_indian}&deceased_died_of_COVID=${benefitsCriteria.deceased_died_of_COVID}&deceased_served_in_active_military=${benefitsCriteria.deceased_served_in_active_military}&shared=true`
+    )
+    const enResults = BENEFITS_ELIBILITY_DATA.scenario_1_covid.en.results
+    pageObjects
+      .benefitsAccordion()
+      .filter(':visible')
+      .should('have.length', enResults.eligible.length)
+      .and('contain', 'Likely Eligible')
+      .and('contain', enResults.eligible.eligible_benefits[0])
+      .and('contain', enResults.eligible.eligible_benefits[1])
+      .and('contain', enResults.eligible.eligible_benefits[2])
+  })
+
+  it('QA scenario 2 Veteran EN - Verify correct benefit results for query values that includes veteran in search parameter of URL', () => {
+    const benefitsCriteria = BENEFITS_ELIBILITY_DATA.scenario_2_veteran.en.param
+    const applicant_date_of_birth = encodeURI(
+      `{"month":"${
+        benefitsCriteria.applicant_date_of_birth_month - 1
+      }","day":"${benefitsCriteria.applicant_date_of_birth_day}","year":"${
+        benefitsCriteria.applicant_date_of_birth_year
+      }"}`
+    )
+    const deceased_date_of_death = encodeURI(
+      `{"month":"${benefitsCriteria.deceased_date_of_death_month - 1}","day":"${
```

Comment:
Please remove this and follow the pattern in scenario3

https://github.com/GSA/px-benefit-finder/pull/902/files#diff-b58f7ff9ab058d9ab02fc76409896fe396763c9930bf33ddc0e2040ad63bdcc3R174-R178

Reply:
I have made the necessary updates